src/service/action-impl.js

```diff
@@ -342,16 +342,16 @@ export class ActionService {
   /**
    * @param {!Element} target
    * @param {string} actionEventType
-   * @return {?{node: !Element, actionInfo: !ActionInfoDef}}
+   * @return {?{node: !Element, actionInfos: Array<!ActionInfoDef>}}
    */
   findAction_(target, actionEventType) {
     // Go from target up the DOM tree and find the applicable action.
     let n = target;
-    let actionInfo = null;
+    let actionInfos = null;
```

Comment:
`matchActionInfos_` returns a nullable value since the result is optional. `actionInfos` could therefore be locally null, but I'll change the scope and make it immutable to avoid null-related errors.